### PR TITLE
feature/add-full-name-to-user-model

### PR DESCRIPTION
**Pull request by rniedziolka**
Hi there,
The full name has been added to employees:
https://www.loom.com/share/d754edf8d7944cedb90662efbe26ab1b
Let me know if you have any suggestions.

### CODE REVIEW

File: app/controllers/admin/employees_controller.rb [hunk 1]
```diff
@@ -15,7 +15,9 @@ def edit
     end
 
     def update
-      if employee.update(employee_params)
+      check_password_params = employee_params
```
Comment (oskarlakner) [inline app/controllers/admin/employees_controller.rb:18]: you could achieve that by using tap inside the employee_params. Tap opens the object and inside the block you can merge / delete from it

File: app/views/employees/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      <ul>
+        <% employee.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field mb-3">
+    <%= form.label :first_name %><br>
```
Comment (oskarlakner) [inline app/views/employees/_form.html.erb:15]: tbh I don't like it - as you are allowing users to edit their first & last name then why not allow them to edit their password? IMO this should be the default devise form

**Review by oskarlakner — APPROVED** [review]

File: app/controllers/admin/employees_controller.rb (hunk 1 above)
Comment (rniedziolka) [reply app/controllers/admin/employees_controller.rb:18]: I'll take a look at it, thanks

File: app/views/employees/_form.html.erb (hunk 2 above)
Comment (rniedziolka) [reply app/views/employees/_form.html.erb:15]: This is only for setting the name once, users are not able to edit it, this is the case in AC

**Pull request merged**